nmrglue/fileio/varian.py

```diff
@@ -175,18 +175,18 @@ def create_pdic_param(name, values):
     Create a fake procpar dictionary element of with given name and values.
     """
     dic = dict()
-    dic["Dgroup"] = '1'
-    dic["Ggroup"] = '2'
-    dic["active"] = '1'
-    dic["basictype"] = '1'
+    dic["Dgroup"] = 1
+    dic["Ggroup"] = 2
+    dic["active"] = 1
+    dic["basictype"] = 1
```

Comment:
If read_procpar does type casting then this value needs to be correct so that the procpar files written out can be read back in.